contract_compassion/model/contracts.py

```diff
@@ -60,6 +60,11 @@ class recurring_contract(models.Model):
         'Partner Contract Number', required=True)
     end_reason = fields.Selection('get_ending_reasons')
     end_date = fields.Date(readonly=True, track_visibility='onchange')
+    paid_months = fields.Integer(

```

Comment:
The field was called `months_paid` and should not be renamed for compatibility with GP.